Update namespace clarity (#33)
rancher-sandbox/docs.rancherdesktop.io: docs/how-to-guides/working-with-images.md

nerdctl -h
```

> Unlike docker, containerd features its own namespaces, by default images are stored in the `default` namespace.

Reviewer: Can we add these statements without removing the earlier statements, as I feel your steps/statements compliment the original information here.

> Unlike docker, containerd features its own namespaces, by default images are stored in the `default` namespace.

Reviewer 2: ```suggestion
Unlike docker, containerd features its own namespaces; by default, images are stored in the `default` namespace.
```

> The Kubernetes images are stored in the `k8s.io` namespace. If you want your images avilable for use by Kubernetes

Reviewer 2: As per Guna's note, if you keep the original verbiage, you can remove the first sentence here.

> The Kubernetes images are stored in the `k8s.io` namespace. If you want your images avilable for use by Kubernetes

Reviewer 2: ```suggestion
If you want your images available for use by Kubernetes,
```

then use the `--namespace k8s.io` or `-n k8s.io` CLI argument.

> nerdctl namespaces are independent and seperate from Kubernetes/kubectl namespaces.

Reviewer 2: ```suggestion
nerdctl namespaces are independent and separate from Kubernetes/kubectl namespaces.
```

```console
nerdctl build -t TAG .
```

> To build an image for use with Kubernetes specify the `k8s.io` namespace.

Reviewer 2: ```suggestion
To build an image for use with Kubernetes, specify the `k8s.io` namespace:
```
